client/.babelrc

```diff
@@ -0,0 +1,15 @@
+{
+    "presets": [
+        "@babel/preset-env",
+        "@babel/preset-react"
+    ],
+    "plugins": [
```

Comment:
Adding this babelrc overrides the presets from NextJS, which is where the 'no need to import react' preset lived. Adding this file is necessary for Styled Components, but just thought I'd mention this is why that broke when you added them (from the looks of the docs).